koku/masu/processor/worker_cache.py

```diff
@@ -72,7 +72,7 @@ def add_task_to_cache(self, task_key):
         task_list = self.host_specific_worker_cache
         task_list.append(task_key)
         self.set_host_specific_task_list(task_list)
-        msg = f"Added {task_key} to cache."
+        msg = f"Added task key {task_key} to Redis cache."
```

Comment:
We actually just switched away from using Redis to caching this in the DB.

Reply:
Oops, didn't know. Let me fix that... :seedling: